Clicker/ViewControllers/PollsViewController.swift

```diff
@@ -188,9 +188,11 @@ class PollsViewController: UIViewController {
     
     // MARK: - View lifecycle
     override func viewWillAppear(_ animated: Bool) {
+        pollsCollectionView.reloadData()
```

Comment:
We actually need this to be `reloadData` rather than `adapter.performUpdates(...)`. If you recall, each of the two cells in the `pollsCollectionView` (created and joined) keep track of their data separately. So, `reloadData` here forces them to send a request to the server for new sessions as we are using `GetPollSessions` in the `configure` of `PollsCell`. 

`adapter.performUpdates(...)` would not work because it would see that the model hasn't changed and nothing would happen, because the model doesn't change until after these cells are reloaded.